Map/RoomBase.gd

```diff
@@ -10,3 +10,15 @@ class_name RoomBase
 
 func _to_string() -> String:
 	return "RoomBase"
+
+func get_room_abbreviation() -> String:
```

Comment:
this might be handled better by a dictionary I think

Reply:
I ended up moving this logic into the Children classes of EventBase.

Reply:
yeah that works nicely too